dockers/docker-orchagent/swssconfig.sh

```diff
@@ -41,6 +41,18 @@ fast_reboot
 
 HWSKU=`sonic-cfggen -d -v "DEVICE_METADATA['localhost']['hwsku']"`
 
+# Don't load json config if system warm start or
+# swss docker warm start is enabled, the data already exists in appDB.
+WARM_START=`redis-cli -n 4 hget "WARM_RESTART|system" enable`
+if [ "$WARM_START" == "true" ]; then
+  exit 0
+fi
+
+WARM_START=`redis-cli -n 4 hget "WARM_RESTART|swss" enable`
+if [ "$WARM_START" == "true" ]; then
+  exit 0
+fi
+
```

Comment:
what if we have modified any of the json?

Reply:
This is a good question.  At restore phase, you get whatever the previous configurations are.

Should anything is to be changed  for any configuration in the json files, probably there should be specific script,  or program implemented for the delta update.   It is similar to configDB data change.

I  doubt this script is the right place to deal with each possible change.  Maybe the long term solution is to eliminate these json files and make them part of the configDB.